pisa/stages/flux/pi_mceq_barr_red.py

```diff
@@ -0,0 +1,489 @@
+"""
+Stage to implement the inclusive neutrino flux as calculated with MCEq,
```

Comment:
update docstring reflecting change that this actually does not do the niominal flux